backend/seafile/seafile_internal_test.go

```diff
@@ -0,0 +1,187 @@
+package seafile
+
+import (
+	"path"
+	"testing"
+)
+
+func TestEmptySplitPathFromRootTarget(t *testing.T) {
+	fs := &Fs{
+		libraryName:   "",
+		rootDirectory: "",
+	}
+	libraryName, directory := fs.splitPath("")
+	if libraryName != "" {
+		t.Errorf("library name should be empty but found '%s'", libraryName)
```

Comment:
In the rclone project we usually use assert and require from testify

So this could become

```go
assert.Equal(t, "", libraryName)
assert.NotEqual(t, "", directory)
```

Most of the time there are no need for comment strings as the error string takes you straight to the code

I'd have put all of the splitSpath tests into one table driven test - maybe like this one

https://github.com/rclone/rclone/blob/74d9dabdff69d7c275db1f09be9767b331d74d4d/fs/sizesuffix_test.go#L15-L34

You can add a `name` parameter to the struct and use an inner `t.Run` if you want like this one for more discoverability

https://github.com/rclone/rclone/blob/74d9dabdff69d7c275db1f09be9767b331d74d4d/fstest/fstests/fstests.go#L573-L597